### PR TITLE
Fix job launch script 

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: scripts/bamboo/build_and_test.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ trycmd "make VERBOSE=1 -j"
 #
 echo "Testing..."
 if [[ $HOSTNAME == *manta* ]] || [[ $HOSTNAME == *ansel* ]]; then
-  lalloc 1 lrun -n 1 ctest --output-on-failure -T Test
```
Comment (mcfadden8) [inline scripts/bamboo/build_and_test.sh]: I'm (too easily) confused again.  Why didn't this work?

Comment (davidbeckingsale) [reply scripts/bamboo/build_and_test.sh]: Let me email you.

Comment (rhornung67) [reply scripts/bamboo/build_and_test.sh]: If this is for CORAL machines, you should do this as lalloc 1 lrun -1 ...  i.e., not '-n 1'

Comment (davidbeckingsale) [reply scripts/bamboo/build_and_test.sh]: We need to avoid `lrun` entirely.

Comment (rhornung67) [reply scripts/bamboo/build_and_test.sh]: Then, you have to submit a batch job I think.

Comment (davidbeckingsale) [reply scripts/bamboo/build_and_test.sh]: Yup, that's what `lalloc` is for. I've got it figured out and tested this already 😄

Comment (rhornung67) [reply scripts/bamboo/build_and_test.sh]: Actually, the manta and ansel systems are different in terms of how you go from a login node to a compute node. ansel has an extra step involving a launch node.

Comment (davidbeckingsale) [reply scripts/bamboo/build_and_test.sh]: Not anymore.

**Pull request merged**